### ISSUE TITLE
USPS returns multiple rates and ignores dimensiosn

### ISSUE THREAD

**Issue by couellet** [kylewest/DotNetShipping#55]
We are specifying dimensions of a package however USPS returns us multiple rates that include the dimension in the name.

For example:

- Priority Mail 3-Days Medium Flat Rate Box
- Priority Mail 3-Days Large Flat Rate Box

However we specified the dimensions in the request and according to their site it should match a Large box.

**Comment by kylewest** [comment]
I'm guessing this is an issue on the USPS side (their API errs on the side of providing more options vs. less). Were the small box, envelope, etc. included in the results?

Kyle

**Comment by couellet** [comment]
Yep all those options were included. We will probably try to filter them on our side. Pretty bad that USPS does not do it out of the box ;)
